sailthru_import/mailchimp-export.py

```diff
@@ -0,0 +1,71 @@
+"""
+This script is deprecated, but can be used as a reference for MailChimp and SailThru API interaction.
```

Comment:
I don't think it makes much sense to check this in, if we're not planning on using it.